### DISCUSSION TITLE
Time difference

### DISCUSSION THREAD

**Discussion by henkiewie** [alexjustesen/speedtest-tracker#457] [Q&A] [answered]
Hi,

Can someone tell me why the webinterface shows a different time +1hour then the setting for the crontab?
<img width="813" alt="Screenshot 2023-03-09 at 07 48 51" src="https://user-images.githubusercontent.com/74558874/223943204-f35e9f76-6230-4f67-b334-37a3123f522e.png">
<img width="1235" alt="Screenshot 2023-03-09 at 07 48 19" src="https://user-images.githubusercontent.com/74558874/223943234-ca25590e-c544-4fbd-98a8-f906eeecc136.png">

regards,
Henk

**Comment by alexjustesen** [comment, marked as answer]
There is some known time/timezone jank happening at the moment. I'm tracking that work under this milestone https://github.com/alexjustesen/speedtest-tracker/milestone/7

**Comment by Drallas** [comment]
Is this related to?

```
        volumes:
            # - '/etc/localtime:/etc/localtime:ro'
```

Together with - TZ=Europe/Amsterdam all Speedtests are 2 hours off.


<img width="1126" alt="Screenshot 2023-08-31 at 13 23 21" src="https://github.com/alexjustesen/speedtest-tracker/assets/24792888/eb478a02-3a44-4432-8761-28e41b5c9a1e">

**Reply by thegodfatherrelish** [reply]
The speed tests are started according to UTC, which means one hour difference to CET (UTC+1) in winter and two hours to CEST (UTC+2) in summer, hence your deviations. I would like to see a function so that the tests are always carried out at the same local time, but I don't know if that would work.
